account_invoice_validation_queued/models/queue_job.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class QueueJob(models.Model):
+    _inherit = 'queue.job'
+
+    def cancel(self):
+        self.filtered(lambda x: x.state in ['pending', 'enqueued']).unlink()
```

Comment:
This can possibly be removed. See comment in view.

Reply:
Yes, but being a public method that can be called from any place, I prefer to filter.